apps/mobile/src/hooks/launchpad/useAddLiquidity.ts

```diff
@@ -0,0 +1,87 @@
+import {AccountInterface, CallData, constants} from 'starknet';
+import {LAUNCHPAD_ADDRESS, EKUBO_DEX_ADDRESS} from 'common';
+
+export const useAddLiquidity = () => {
+  const addLiquidityUnrug = async (
+    account: AccountInterface,
+    coinAddress: string,
+    amount: string,
+    startingPrice: string,
+    lockTime: string
+  ) => {
+    try {
+      const deployCall = {
+        contractAddress: LAUNCHPAD_ADDRESS[constants.StarknetChainId.SN_SEPOLIA],
+        entrypoint: 'add_liquidity_unrug',
+        calldata: CallData.compile({
```

Comment:
Not the correct input, params etc of the launchpad.cairo, but thank you let me know on it later